zipline/utils/tradingcalendar.py

```diff
@@ -403,3 +403,9 @@ def get_open_and_closes(trading_days, early_closes, get_open_and_close):
 
 open_and_closes = get_open_and_closes(trading_days, early_closes,
                                       get_open_and_close)
+
+
+def market_minutes_for_day(stamp):
+    index = open_and_closes.index.get_loc(stamp.date())
```

Comment:
There's a different `get_open_and_close` in `tradingcalendar` already, which is what actually figures out the opens and closes, as opposed to just reading them. Maybe we could rename one of them?